app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -10,11 +10,18 @@ class ParticipantDeclarationsController < Api::ApiController
       include ApiFilter
 
       def create
-        params = HashWithIndifferentAccess.new({ cpd_lead_provider: }).merge(permitted_params["attributes"] || {})
+        if permitted_params.dig("attributes", "declaration_type")
+          service = RecordDeclarations::Started::EarlyCareerTeacher
+                      .new(permitted_params["attributes"].merge(cpd_lead_provider:))
 
-        log_schema_validation_results
+          render_from_service(service, ParticipantDeclarationSerializer)
+        else
```

Comment:
why do we want/need this `else` branch? are we not able to delegate to the service?